### ISSUE TITLE
Request for Conda Installation Support for fastpdb

### ISSUE THREAD

**Issue by zhu0619** [biotite-dev/fastpdb#15]
Thank you for developing fastpdb. It's a fantastic tool!

Would it be possible to add conda installation support? Many users, including myself, rely on conda for managing environments, and it would make fastpdb more accessible.

I’m happy to assist or test if needed.

**Comment by padix-key** [comment]
Good idea, especially since all other packages from [biotite-dev](https://github.com/biotite-dev/) are also on Conda. I created a PR: https://github.com/conda-forge/staged-recipes/pull/27246

**Comment by padix-key** [comment]
Currently the build fails due to a hickup with the NumPy version. Probably the easiest way to solve this is to wait until Biotite 1.0 is released in the upcoming weeks.

**Comment by zhu0619** [comment]
Thank you for addressing this so quickly. Looking forward to the new release.

**Comment by cwognum** [comment]
Hi @padix-key, now that https://github.com/biotite-dev/biotite/pull/635 is merged and v1 is released, are there any other blockers for this issue?

**Comment by padix-key** [comment]
Previously this was blocked by issues to build Biotite 1.0 on Conda. With Biotite `1.0.1` this is resolved. I already added a Conda recipe for `fastpdb` to `staged-recipes` (https://github.com/conda-forge/staged-recipes/pull/27246).

**Comment by padix-key** [comment]
Implemented by https://github.com/conda-forge/staged-recipes/pull/27246
